### PR TITLE
Multi: fix "Is Legacy" control misalignment, set its default value to checked, and make it persistent

### LINKED ISSUE #2784: VSP "Is Legacy" checkbox moves when it is checked
Peek 2020-10-24 10-00

### LINKED ISSUE #2785: Is Legacy should be checked by default
The new and legacy VSP forms are very similar, and when you open the Tickets tab its not immediately obvious which one you are looking at.

In order to keep a smooth UX for users of legacys VSPs (ie. everybody right now) - the legacy VSP form should be the default, and switching to the new VSP form should be a conscious choice.

As a bonus - decrediton should remember which one you had open most recently. Eg. If I buy legacy tickets, next time I open decrediton it should open the legacy form.

### PR DESCRIPTION
**Pull request by bgptr**
Closes #2785 
Closes #2784

### CODE REVIEW

File: app/actions/VSPActions.js [hunk 1]
```diff
@@ -467,6 +467,8 @@ export const discoverAvailableStakepools = () => async (dispatch, getState) => {
 
 export const TOGGLE_ISLEGACY = "TOGGLE_ISLEGACY";
 export const toggleIsLegacy = (isLegacy) => (dispatch) => {
+  const config = getGlobalCfg();
```
Comment (vctt94) [inline app/actions/VSPActions.js:470]: wallet config

**Review by vctt94 — CHANGES_REQUESTED** [review]
Same of #2789

**Comment by bgptr** [comment]
Thank you for thre review. 
I moved the isLegacy param to the wallet config.

File: app/config.js [hunk 2]
```diff
@@ -55,6 +55,9 @@ export function initWalletCfg(testnet, walletPath) {
   if (!config.has("trezor")) {
     config.set("trezor", false);
   }
+  if (!config.has("vsp_is_legacy")) {
+    config.set("vsp_is_legacy", true);
```
Comment (vctt94) [inline app/config.js:59]: Thinking if this should be true or false. 

What do you think @alexlyp ?

**Review by vctt94 — COMMENTED** [review]
Looks good, but I am not sure if the ticket default value should be legacy or not.

I'd rather wait on ay-p's opinion

File: app/config.js (hunk 2 above)
Comment (alexlyp) [reply app/config.js:59]: We should be defaulting to use the new VSP ticket purchasing system. (non-legacy)

Comment (bgptr) [reply app/config.js]: Alright. I've changed it to false and resolved the merge conflicts.

**Review by vctt94 — APPROVED** [review]
LGTM

**Pull request merged**